certified-connectors/Feathery Forms/Readme.md

```diff
@@ -0,0 +1,12 @@
+## Feathery Forms Connector
```

Comment:
Please update readme.md file as per the template below. 
https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md

Reply:
this has been updated!